pages/letushelp/prearraignment/index.tsx

```diff
@@ -0,0 +1,76 @@
+import Head from 'next/head'
+import { copy } from '../../../constants/prearraignment-data'
+import { siteTitle } from '../../../constants/copy'
+import Grid from '@material-ui/core/Grid'
+import useLanguage from '../../../hooks/useLanguage'
+import { Title, Paragraph } from '../../../ui'
+import { useStyles } from '../../../constants/materialStyles'
+const PreArraignmentPage = () => {
+  const { language } = useLanguage()
+  const classes = useStyles()
+  const activeCopy = copy[language]
+  return (
+    <>
+      <Head>
+        <title>{`${siteTitle} | ${activeCopy.title}`}</title>
+      </Head>
+      <div style={{ margin: '6rem 0 2rem 0' }}>
+        <Grid container spacing={3}>
+          <Grid item xs={11} sm={2}>
+            <img
+              className={classes.arraignIcons}
+              src={'/icons/prearraignment.svg'}
+              alt="Pre-arraignment icon"
+            />
+          </Grid>
+          <Grid item xs={11} sm={10}>
+            <Title className={classes.arraignTextCenter}>
+              {activeCopy.title}{' '}
+            </Title>
+            <h2 className={classes.arraignTextCenter}>{activeCopy.subtitle}</h2>
+            <Paragraph className={classes.arraignP} size="med-text">
+              {activeCopy.description}
+            </Paragraph>
+          </Grid>
+          <hr
+            style={{
+              border: '0',
+              clear: 'both',
+              display: 'block',
+              width: '100%',
+              backgroundColor: '#12385e',
+              height: '2px',
+            }}
+          />
+          <Grid item xs={11} sm={3}>
+            <img
```

Comment:
Still getting some things appearing off center for the icons in the mobile views.

I think you're gonna get a better result using [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) to center these instead of using margins. It will allow your layouts to be more adaptive.
 
![Screen Shot 2021-05-12 at 4 45 48 PM](https://user-images.githubusercontent.com/72828520/118057883-b96b5f80-b341-11eb-9615-ef31a9c995f1.png)